extensions/wikia/Forum/ForumSpecialController.class.php

```diff
@@ -59,24 +59,27 @@ public function index() {
 
 		$output->setPageTitle( wfMessage( 'forum-forum-title' )->plain() );
 
-
-		$this->response->setVal('blurb', wfMessage( 'forum-specialpage-blurb' )->parse());
-		$this->response->setVal('blurbHeading', wfMessage( 'forum-specialpage-blurb-heading' )->parse());
-		$this->response->setVal('lastPostByMsg', wfMessage( 'forum-specialpage-board-lastpostby' )->escaped());
-		$this->response->setVal('canEdit', $this->wg->User->isAllowed( 'forumadmin' ));
-		$this->response->setVal('editUrl', $this->wg->Title->getFullUrl( 'action=editmode' ));
+		$this->blurb = wfMessage( 'forum-specialpage-blurb' )->parse();
+		$this->blurbHeading = wfMessage( 'forum-specialpage-blurb-heading' )->parse();
+		$this->lastPostByMsg = wfMessage( 'forum-specialpage-board-lastpostby' )->escaped();
+		$this->canEdit = $this->wg->User->isAllowed( 'forumadmin' );
+		$this->editUrl = $this->wg->Title->getFullUrl( 'action=editmode' );
 
 		$forum = new Forum();
 
 		// TODO: once we're sure the forums are properly created when importing starter wiki
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)